executor/executor_test.go

```diff
@@ -66,6 +67,7 @@ import (
 	"github.com/pingcap/tidb/util/testutil"
 	"github.com/pingcap/tidb/util/timeutil"
 	"github.com/pingcap/tipb/go-tipb"
+	"github.com/tiancaiamao/debugger"
```

Comment:
Could we remove this package to tidb？

Reply:
IMHO, it's better for this PR to focus on bug fix only.
Maybe I could clean up here and move the package to TiDB in a seperate PR.